mlflow/sagemaker/__init__.py

```diff
@@ -360,15 +359,10 @@ def _deploy(
     if endpoint_exists and mode == DEPLOYMENT_MODE_CREATE:
         raise MlflowException(
             message=(
-                "You are attempting to deploy an application with name: {application_name} in"
-                " '{mode_create}' mode. However, an application with the same name already"
-                " exists. If you want to update this application, deploy in '{mode_add}' or"
-                " '{mode_replace}' mode.".format(
-                    application_name=app_name,
-                    mode_create=DEPLOYMENT_MODE_CREATE,
-                    mode_add=DEPLOYMENT_MODE_ADD,
-                    mode_replace=DEPLOYMENT_MODE_REPLACE,
-                )
+                f"You are attempting to deploy an application with name: {app_name} in"
+                f" '{DEPLOYMENT_MODE_CREATE}' mode. However, an application with the same name"
+                f" already exists. If you want to update this application, deploy in"
```

Comment:
```suggestion
                " already exists. If you want to update this application, deploy in"
```

Reply:
Thank you for your checking. Sorry for a lot of mistakes :(